apps/entry/filter_set.py

```diff
@@ -643,6 +645,12 @@ def lead_group_label_filter(self, queryset, name, value):
             return queryset.filter(entrygrouplabel__group__title__icontains=value)
         return queryset
 
+    def lead_has_assessment_filter(self, qs, _, value):
+        if value is None:
+            return qs
+        # TODO: We need to add new assessment module filter here after it is deployed
+        return qs.filter(lead__assessment__isnull=not value)
```

Comment:
@Rup-Narayan-Rajbanshi 
We will need to update this to new ARY. Make sure to update this before new assessment merge.